editor/plugins/node_3d_editor_plugin.h

```diff
@@ -123,6 +123,7 @@ class Node3DEditorViewport : public Control {
 		VIEW_HALF_RESOLUTION,
 		VIEW_AUDIO_LISTENER,
 		VIEW_AUDIO_DOPPLER,
+		VIEW_GRID,
 		VIEW_GIZMOS,
```

Comment:
Nitpick, but this would make it match the order in which it's registered / handled in the .cpp, and shown in the UI.
```suggestion
		VIEW_GIZMOS,
		VIEW_GRID,
```